src/DssSpell.sol

```diff
@@ -60,6 +140,130 @@ contract DssSpellAction is DssAction {
+        JugAbstract jug = JugAbstract(DssExecLib.jug());
+        CureLike cure = CureLike(DssExecLib.getChangelogAddress("MCD_CURE"));
+        address dai = DssExecLib.dai();
+        IlkRegistryAbstract ilkRegistry = IlkRegistryAbstract(DssExecLib.reg());
+        address esm = DssExecLib.esm();
+
+        // Run sanity checks
+        require(TeleportJoinLike(TELEPORT_JOIN).vat() == address(vat));
+        require(TeleportJoinLike(TELEPORT_JOIN).daiJoin() ==  DssExecLib.daiJoin());
+        require(TeleportJoinLike(TELEPORT_JOIN).ilk() == ILK);
+        require(TeleportJoinLike(TELEPORT_JOIN).domain() == DOMAIN_ETH);
+        require(TeleportOracleAuthLike(ORACLE_AUTH).teleportJoin() == TELEPORT_JOIN);
+        require(TeleportRouterLike(ROUTER).dai() == dai);
+        require(TeleportFeeLike(LINEAR_FEE).fee() == WAD / 10000);
+        require(TeleportFeeLike(LINEAR_FEE).ttl() == 8 days);
+        require(OptimismTeleportBridgeLike(TELEPORT_GATEWAY_OPT).l1Escrow() == ESCROW_OPT);
+        require(OptimismTeleportBridgeLike(TELEPORT_GATEWAY_OPT).l1TeleportRouter() == ROUTER);
+        require(OptimismTeleportBridgeLike(TELEPORT_GATEWAY_OPT).l1Token() == dai);
+        require(OptimismTeleportBridgeLike(TELEPORT_GATEWAY_OPT).l2TeleportGateway() == TELEPORT_L2_GATEWAY_OPT);
+        require(OptimismTeleportBridgeLike(TELEPORT_GATEWAY_OPT).messenger() == MESSENGER_OPT);
+        require(ArbitrumTeleportBridgeLike(TELEPORT_GATEWAY_ARB).l1Escrow() == ESCROW_ARB);
+        require(ArbitrumTeleportBridgeLike(TELEPORT_GATEWAY_ARB).l1TeleportRouter() == ROUTER);
+        require(ArbitrumTeleportBridgeLike(TELEPORT_GATEWAY_ARB).l1Token() == dai);
+        require(ArbitrumTeleportBridgeLike(TELEPORT_GATEWAY_ARB).l2TeleportGateway() == TELEPORT_L2_GATEWAY_ARB);
+        require(ArbitrumTeleportBridgeLike(TELEPORT_GATEWAY_ARB).inbox() == INBOX_ARB);
+
+        vat.init(ILK);
+        jug.init(ILK);
+
+        DssExecLib.increaseGlobalDebtCeiling(2_000_000);
+        DssExecLib.setIlkDebtCeiling(ILK, 2_000_000);
+        vat.file(ILK, "spot", RAY);
+
+        cure.lift(TELEPORT_JOIN);
+
+        vat.rely(TELEPORT_JOIN);
+
+        // Configure TeleportJoin
+        TeleportJoinLike(TELEPORT_JOIN).rely(ORACLE_AUTH);
+        TeleportJoinLike(TELEPORT_JOIN).rely(ROUTER);
+        TeleportJoinLike(TELEPORT_JOIN).rely(esm);
+
+        TeleportJoinLike(TELEPORT_JOIN).file("vow", DssExecLib.vow());
+
+        TeleportJoinLike(TELEPORT_JOIN).file("fees", DOMAIN_OPT, LINEAR_FEE);
+        TeleportJoinLike(TELEPORT_JOIN).file("line", DOMAIN_OPT, 1_000_000 * WAD);
+
+        TeleportJoinLike(TELEPORT_JOIN).file("fees", DOMAIN_ARB, LINEAR_FEE);
+        TeleportJoinLike(TELEPORT_JOIN).file("line", DOMAIN_ARB, 1_000_000 * WAD);
+
+        // Configure TeleportOracleAuth
+        TeleportOracleAuthLike(ORACLE_AUTH).rely(esm);
+
+        TeleportOracleAuthLike(ORACLE_AUTH).file("threshold", 13);
+        address[] memory oracles = new address[](24);
```

Comment:
Consider adding the link here:
https://forum.makerdao.com/t/maker-teleport-oracle-launch-configuration/17471